patches/snappy-force-load-aa-profile.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/daemon/apparmor_default.go b/daemon/apparmor_default.go
+index 2a418b2..fa9fe5b 100644
+--- a/daemon/apparmor_default.go
++++ b/daemon/apparmor_default.go
+@@ -4,6 +4,7 @@ package daemon
+ 
+ import (
+ 	"fmt"
++	"os"
+ 
+ 	aaprofile "github.com/docker/docker/profiles/apparmor"
+ 	"github.com/opencontainers/runc/libcontainer/apparmor"
+@@ -22,7 +23,7 @@ func ensureDefaultAppArmorProfile() error {
+ 		}
+ 
+ 		// Nothing to do.
+-		if loaded {
++		if loaded && os.Getenv("DOCKER_FORCE_LOAD_AA_PROFILE") != "1" {
```

Comment:
Thanks.
I was thinking to shorten DOCKER_FORCE_LOAD_AA_PROFILE as well since the env var I'm using right is a little bit longer.  It looked not so good when I tried to wrap the long line after gofmt